src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -374,6 +375,7 @@ def update_grids_sites_table(self, dataframe: pd.DataFrame, table=None) -> None:
             table = self.grids_sites_table
 
         dataframe.reset_index(drop=True, inplace=True)
+        up_to_date_data["last_updated"] = datetime.now(timezone.utc)
```

Ruff (0.8.2)

Undefined name `up_to_date_data`

(F821)